Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -74,6 +74,13 @@ PermitRootLogin No
 UsePrivilegeSeparation sandbox
 </source>
 
+File: <code>/etc/ssh/moduli</code>
+
+All Diffie-Hellman moduli in use should be at least 3072-bit-long (they are used for <code>diffie-hellman-group-exchange-sha256</code>). From the structure of <code>moduli</code> files, this means the fifth field of all lines in this file should be greater than or equal to 3071.
```

Comment:
would rephrase a bit for clarity & conciseness (same for subsequent paragraph):

> All Diffie-Hellman moduli in use should be at least 3072-bit-long (they are used for <code>diffie-hellman-group-exchange-sha256</code>) as per our [[Security/Guidelines/Key_Management]] recommendations.


(cut rest of paragraph)